packages/webapp/src/utils/application.js

```diff
@@ -187,7 +187,7 @@ const getHabitats = session => {
 
   const proposed = proposedIdentifiers.flatMap(identifier =>
     metricData[identifier].filter(details => 'Condition' in details).map(details => ({
-      proposedHabitatId: details['Habitat reference Number'],
+      proposedHabitatId: details['Habitat reference Number'] ? String(details['Habitat reference Number']) : details['Habitat reference Number'],
```

Comment:
```suggestion
      proposedHabitatId: details['Habitat reference Number'] ?? String(details['Habitat reference Number']),
```

does this work?

Reply:
Suggested change won't convert from number to string unfortunately, but the original will